modules/deploy/pages/deployment-option/cloud/managed-connectors/converters-and-serialization.adoc

```diff
@@ -2,13 +2,23 @@
 :description: Use converters to handle the serialization and deserialization of data between a Redpanda topic and a managed connector.
 :page-cloud: true
 
-Converters handle the serialization and deserialization of data between a Redpanda topic and a connector. 
+Connectors are a translation layer working between the Redpanda and the remote system. 
```

Comment:
```suggestion
Connectors are a translation layer working between Redpanda and the remote system. 
```

Reply:
Perhaps:
Connectors are a translation layer working between Redpanda and a remote system.